packages/sdk-size-test/app/routes/index.tsx

```diff
@@ -0,0 +1,12 @@
+import type { LoaderFunction, MetaFunction } from "@remix-run/node";
+import CatchAllContnet, {
+  loader as catchAllloader,
+  meta as catchAllmeta,
+  links as catchAllLinks,
+} from "./$";
+
+export const meta: MetaFunction = (args) => catchAllmeta(args);
+export const loader: LoaderFunction = (args) => catchAllloader(args);
+export const links = catchAllLinks;
+const Content = () => <CatchAllContnet />;
```

Comment:
why all that for simple test?

Reply:
To be as much as near to real app.

Reply:
maybe you want to use an actual bundler in that case, the same one we use for deployment, I don't really want us to maintain this setup just to test the size, especially if it can become different than our actual deployment